@here/harp-examples/src/datasource_webtile.ts

```diff
@@ -3,21 +3,17 @@
  * Licensed under Apache 2.0, see full license in LICENSE
  * SPDX-License-Identifier: Apache-2.0
  */
-
 import { MapControls, MapControlsUI } from "@here/harp-map-controls";
-import { CopyrightElementHandler, MapView } from "@here/harp-mapview";
+import { CopyrightElementHandler, CopyrightInfo, MapView, Tile } from "@here/harp-mapview";
 import { WebTileDataSource } from "@here/harp-webtile-datasource";
-import { apikey } from "../config";
+import THREE = require("three");
 
 // tslint:disable:max-line-length
 /**
- * A simple example using the webtile data source. Tiles are retrieved from
- * ```
- * https://1.base.maps.ls.hereapi.com/maptile/2.1/maptile/newest/normal.day/${level}/${column}/${row}/512/png8?apikey=${apikey}
- * ```
+ * A simple example using the webtile data source. Tiles are generated from a
```

Comment:
this example should be renamed, because `WebTile` may have people think they expect satellite images, how about `CustomTextureDataSourceExample`.. or some equivalent?

Reply:
done